athorn/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> Anthony Thorn </title>
+    <link href ="css/style.css" rel ="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Reem+Kufi" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lemonada" rel="stylesheet">
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+  </head>
+
+  <body>
+
+    <div class ="nav">
+      <ul>
+        <li class ="nav-link"><a href="index.html"> Home </a></li>
+        <li class ="nav-link"><a href="Gallery.html"> Gallery </a></li>
+        <li class ="nav-link"><a href="http://saskartists.ca/"> Sask Artists </a></li>
+      </ul>
+    </div>
+
+    <div class="container">
+
+      <img src ="images/blackwhite.jpg" alt ="Anthony Thorn in Black and White" class ="imageleft"/>
+      <img src ="images/color.jpg" alt ="Anthony Thorn in color" class ="imageright"/>
+
+    </div>
+
+    <div class ="info">
+
+      <h1 class="titleinfo"> About Anthony Thorn </h1>
+      <p class="bio"> Anthony Thorn, born Arthur Goldman, is a prominent Saskatchewan artist from Regina born in 1927.
+        <br> Thorn began painting in 1948 shortly after graduating from the University of Saskatchewan with a Bachelor of Arts.
+        <br> with a Bachelor of Arts. He had nearly 1200 original wokrks in his collection using the mediums of drawing, oil
+        <br> painting, sculpture, and watercolour. Thorn was a founding member of the Saskatchewan Society of Artists and has
```

Comment:
I was more asking about the fact that there are so many line breaks. Why are they there? Doesn't the text automatically wrap?